src/Grids/nodes_and_spacings.jl

```diff
@@ -233,10 +232,23 @@ julia> zspacings(grid, Center(), Center(), Center())
 destantiate(::Face)   = Face
 destantiate(::Center) = Center
 
-function minimum_spacing(dir, grid, ℓx, ℓy, ℓz)
-    spacing = eval(Symbol(dir, :spacing))
+function minimum_spacing(::Val{:x}, grid, ℓx, ℓy, ℓz)
+    LX, LY, LZ = map(destantiate, (ℓx, ℓy, ℓz))
+    Δ = KernelFunctionOperation{LX, LY, LZ}(xpacing, grid, ℓx, ℓy, ℓz)
```

Comment:
```suggestion
    Δ = KernelFunctionOperation{LX, LY, LZ}(xspacing, grid, ℓx, ℓy, ℓz)
```

I also think you could have done `spacing(::Val{:x}) = xspacing` instead of duplicating the whole code.

Reply:
Or do `getglobal(@__module__, Symbol(dir, spacing))`